spec/Classes.tex

```diff
@@ -186,17 +211,39 @@ \subsection{Class Types}
 of other class declarations.
 
 If no memory management strategy is indicated, \chpl{borrowed} will be
-used. The memory management strategies have the following meaning:
+used.
+
+\begin{rationale}
+  \chpl{borrowed} is the default memory management strategy for a class
+  type because there are likely to be more variables referring to a class
+  instance than managing its memory. A further advantage of this default
+  is that \chpl{borrowed}, along with \chpl{unmanaged}, has minimal
+  performance overhead.
+\end{rationale}
+
+The memory management strategies have the following meaning:
 
 \begin{itemize}
-  \item \chpl{borrowed} refers to a class instance that has a lifetime
-    managed by another variable.
-  \item \chpl{unmanaged} the instance must have \chpl{delete} called on it explicitly.
   \item \chpl{owned} the instance will be deleted
     automatically when the \chpl{owned} variable goes out of scope, but
     only one \chpl{owned} variable can refer to the instance at a time.
+    See
+    \\ %formatting
+    \mbox{$$ $$ $$ $$ $$} %indent
+    \url{https://chapel-lang.org/docs/builtins/internal/OwnedObject.html}
+
   \item \chpl{shared} will be deleted when all of the \chpl{shared}
     variables referring to the instance go out of scope.
+    See
+    \\ %formatting
+    \mbox{$$ $$ $$ $$ $$} %indent
+    \url{https://chapel-lang.org/docs/builtins/internal/SharedObject.html}.
+
+  \item \chpl{borrowed} refers to a class instance that has a lifetime
+    managed by another variable.
+
+  \item \chpl{unmanaged} the instance must have \chpl{delete} called on it explicitly.
```

Comment:
Let us add "to [free|reclaim|whatever] its memory" (as you have done in other places).

Because, technically, it is still correct not to call `delete`.